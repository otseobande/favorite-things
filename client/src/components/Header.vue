<template>
  <div class="container pt-4 text-lg text-white">
    <h2 class="sticky left-0 pl-2 font-extrabold text-2xl inline">
      Favorite Things! <img src="../assets/happy.svg" width="30" class="inline"/>
    </h2>
  </div>
</template>

<script>
export default {

}
</script>
