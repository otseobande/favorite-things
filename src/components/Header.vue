<template>
  <div class="container mx-auto pt-4 text-lg text-white">
    <h2 class="font-extrabold text-2xl">Favorite Things!</h2>
  </div>
</template>

<script>
export default {

}
</script>
