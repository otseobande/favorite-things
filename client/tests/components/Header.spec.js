import { mount } from '@vue/test-utils';
import Header from '../../src/components/Header.vue';

describe('Header component', () => {
  it('should match snapshot', () => {
    const wrapper = mount(Header);

    expect(wrapper).toMatchSnapshot();
  });
});
