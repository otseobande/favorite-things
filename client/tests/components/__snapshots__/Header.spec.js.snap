// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div class="container pt-4 text-lg text-white">
  <h2 class="sticky left-0 pl-2 font-extrabold text-2xl inline">
    Favorite Things! <img src="../assets/happy.svg" width="30" class="inline"></h2>
</div>
`;
